### ISSUE TITLE
Accessibility issue with YouTube video element/

### ISSUE THREAD

**Issue by rburch** [contao/contao#7656]
### Affected version(s)

5.3.15

### Description

I noticed that iframes pulled in when doing a Youtube video element are missing titles and there's no field to put one in. This breaks W3C accessibility standards. I didn't check, but my hunch is older Contao versions have the same issue, and likely the Vimeo elements as well.

**Comment by fritzmg** [comment]
See https://github.com/contao/contao/pull/7572 :)
